src/grid/index.js

```diff
@@ -42,9 +43,7 @@ const Grid = ( {
 					<div className="stc-grid-item" key={ index }>
 						<div className="stc-grid-item-inner">
 							{ item.badge ? (
-								<div className="stc-grid-item-badge">
-									{ item.badge }
-								</div>
+								<PremiumBadge badge={ item.badge } />
```

Comment:
Actually, We can create the `<Badge>` component and add the `premium` as a variation.

Same as `<Button>` component with `primary` variation.